release-note-script/src/main/java/ReleaseNoteCreation.java

```diff
@@ -219,30 +219,33 @@ private boolean isContainedInSameAsItems(ReleaseNoteText rnText) {
   }
 
   public enum Category {
-    ENHANCEMENT("Enhancements"),
-    IMPROVEMENT("Improvements"),
-    BUGFIX("Bug fixes"),
-    DOCUMENTATION("Documentation"),
-    MISCELLANEOUS("Miscellaneous");
+    BACKWARD_INCOMPATIBLE("Backward incompatibles", "backward-incompatible"),
```

Comment:
I'll change the display name from `Backward incompatibles` to `Backward-incompatible changes` since I got a request from @josh-wong

Reply:
Done. at dabd278